docs/contributing/setting-up-project.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Setting up the Project for Contributors
+---
+Tach CLI is an open source 
+
+#### 1. Fork the repository
+
+#### 2. Clone the repository locally
+```bash 
+git clone https://github.com/github_username/tach.git
+cd tach
+```
+It is adviced to work on a branch of your own and then raise a PR
+```bash
+git checkout -b "your_feature_name" 
+```
+
+#### 3. Create a virtual environment
+```bash
+python3 -m venv env
+```
+Activate the environment 
+```bash
+source env/Scripts/activate
+```
```

Comment:
These sections should be removed, since we should assume open-source contributors are able to fork the repo and open a pull request. There might be a standard GitHub documentation link we can provide, but we shouldn't need to explain it from scratch.

For the virtual environment, I'll add more detail in other comments but this is managed automatically by our `Makefile`

Reply:
even the dev-requirements is handled in Makefile, removing it from the doc